Microsoft.Quantum.Canon/Utils/Multiplexor.qs

```diff
@@ -0,0 +1,285 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace Microsoft.Quantum.Canon
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Extensions.Math;
+
+    /// # Summary
+    /// Multiply-controlled unitary operation $U$ that performs rotations
```

Comment:
It's not clear from this summary that `MultiplexorPauli` applies the given multiplexed unitary. Could we make that explicit in the summary and change the name to use a verb (e.g.: `MultiplexPauli`)?

Reply:
Renamed nouns to verbs where appropriate.